spire/templates/apps/dovetail-router.yml

```diff
@@ -739,6 +741,25 @@ Resources:
             Effect: Allow
             Principal:
               Service: ecs-tasks.amazonaws.com
+          - Action:
+              - dynamodb:BatchGetItem
+              - dynamodb:ConditionCheck
+              - dynamodb:DescribeTable
+              - dynamodb:DescribeTimeToLive
+              - dynamodb:GetItem
+              - dynamodb:Query
+            Effect: Allow
+            Resource: !Split
+              - ","
+              - Fn::Sub:
+                  - arn:aws:dynamodb:*:*:table/${inner}
+                  - inner:
+                      Fn::Join:
+                        - ",arn:aws:dynamodb:*:*:table/"
+                        - !Split [",", !Ref FrequencyDynamodbTableName]
+          - Action: sts:AssumeRole
+            Effect: Allow
+            Resource: !Split [",", !Ref FrequencyDynamodbAccessRoleArn]
```

Comment:
Similar situation here: I believe we expect this to be a single role ARN now, so no need to `Split`.